mappings/net/minecraft/server/command/RecipeCommand.mapping

```diff
@@ -1,6 +1,13 @@
 CLASS sw net/minecraft/server/command/RecipeCommand
 	FIELD a GIVE_FAILED_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
 	FIELD b TAKE_FAILED_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
-	METHOD a give (Lcd;Ljava/util/Collection;Ljava/util/Collection;)I
+	METHOD a executeGive (Lcd;Ljava/util/Collection;Ljava/util/Collection;)I
+		ARG 0 source
+		ARG 1 targets
+		ARG 2 receoies
 	METHOD a register (Lcom/mojang/brigadier/CommandDispatcher;)V
-	METHOD b take (Lcd;Ljava/util/Collection;Ljava/util/Collection;)I
+		ARG 0 dispatcher
+	METHOD b executeTake (Lcd;Ljava/util/Collection;Ljava/util/Collection;)I
+		ARG 0 rsource
+		ARG 1 targets
+		ARG 2 recepies
```

Comment:
Oh, `recipies`?

Reply:
I managed to confuse you to the point where you don't know the spelling either anymore :D `recipes`